spec/support/shared/shopify_shop.rb

```diff
@@ -0,0 +1,20 @@
+RSpec.shared_context 'shopify_shop' do
+  let(:refund_amount) { 100 }
+  let(:shopify_payment_method) do
+    gateway = Spree::Gateway::ShopifyGateway.new
+    gateway.set_preference(:api_key, ENV.fetch('SHOPIFY_API_KEY'))
+    gateway.set_preference(:password, ENV.fetch('SHOPIFY_PASSWORD'))
+    gateway.set_preference(:shop_name, ENV.fetch('SHOPIFY_SHOP_NAME'))
+    gateway
+  end
+
+  def mock_request(endpoint, extension)
```

Comment:
I feel like it may have something to do with the overall [test harness setup surrounding](http://stackoverflow.com/questions/10545162/undefined-method-path-for-nilnilclass-using-chargify-api-ares-gem) the Shopify configuration.